### PR TITLE
Improve Default Config

### PR DESCRIPTION
**Pull request by Sprungente**
Hope it is helpful. Feel free to tell something. Made with love ❤️

### CODE REVIEW

File: advanced-achievements-plugin/src/main/resources/config.yml [hunk 1]
```diff
@@ -8,8 +8,21 @@
 # Only spaces, NO tabs. Don't forget the indents (2 spaces per indentation level).
 # To check syntax errors in your file, use: yaml-online-parser.appspot.com
 # Default config.yml: github.com/PyvesB/AdvancedAchievements/blob/master/advanced-achievements-plugin/src/main/resources/config.yml
-# Visit the wiki for some valuable additional documentation: github.com/PyvesB/AdvancedAchievements/wiki
 
+# If you want to help Advanced Achievements, so feel free to check our github:#
```
Comment (PyvesB) [inline advanced-achievements-plugin/src/main/resources/config.yml]: The "so" is not needed in this sentence ^^

File: advanced-achievements-plugin/src/main/resources/config.yml [hunk 2]
```diff
@@ -331,22 +350,25 @@ Kills:
...
-      Message: You killed a Mythic Mob!
+      Goal: "Kill the evil Skeletal Knight."
+      Message: "You killed a Mythic Mob!"
       DisplayName: Mythic Killer
       Name: kills_1_skeletal_night
   specificplayer-56c79b19-4500-466c-94ea-514a755fdd09:
     1:
-      Message: You killed the author of the plugin!
-      DisplayName: No More DarkPyves
+      Goal: "Kill the evil Author."
```
Comment (PyvesB) [inline advanced-achievements-plugin/src/main/resources/config.yml]: 😄

File: advanced-achievements-plugin/src/main/resources/config.yml [hunk 3]
```diff
@@ -469,135 +504,154 @@ Milk:
...
+    Goal: "Trade with a villager."
+    Message: "Doing business with your first villager!"
     Name: trades_1
     DisplayName: First Trade    
     
 # When an anvil is used.
 AnvilsUsed:
   1:
-    Message: You just repaired your first item!
+    Goal: "Rapair an item."
```
Comment (PyvesB) [inline advanced-achievements-plugin/src/main/resources/config.yml]: Typo: repair

File: advanced-achievements-plugin/src/main/resources/config.yml [hunk 4]
```diff
@@ -469,135 +504,154 @@ Milk:
...
+    Goal: "Drink 100 Potions"
+    Message: "Drinking weird beverages all the time..."
     Name: consumedpotions_100
     DisplayName: Alcoholic 
     
 # Amount of time played on the server (in hours, use integers).
 PlayedTime:
   1:
-    Message: You played for one hour on the server!
+    Goal: "Play a hour"
```
Comment (PyvesB) [inline advanced-achievements-plugin/src/main/resources/config.yml]: Typo: an hour

File: advanced-achievements-plugin/src/main/resources/config.yml [hunk 5]
```diff
@@ -401,14 +427,16 @@ Deaths:
...
+    Goal: "shoot 50 arrows."
+    Message: "50 arrows shot!"
     Name: arrows_50
     DisplayName: Arrows Around the Place
     
 # When a snowball is thrown.  
 Snowballs:
   1000:
-    Message: 1000 snowballs thrown. That's the spirit!
+    Goal: "throw 1000 Snowbaals."
```
Comment (PyvesB) [inline advanced-achievements-plugin/src/main/resources/config.yml]: Typo: snowballs

**Comment by Sprungente** [comment]
Really? damn^^ 
I think it is easier when """ are there because it's easier to copy and suggest another to using colors or something. I hated to add color and the """ after pasting achievements^^

File: advanced-achievements-plugin/src/main/resources/config.yml (hunk 3 above)
Comment (Sprungente) [reply advanced-achievements-plugin/src/main/resources/config.yml]: oops^^

File: advanced-achievements-plugin/src/main/resources/config.yml (hunk 5 above)
Comment (Sprungente) [reply advanced-achievements-plugin/src/main/resources/config.yml]: damn so many mistakes :O

**Pull request merged**